src/ui/components/radioSelector/RadioSelector.native.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { fireEvent, render, screen } from 'tests/utils'
+
+import { RadioSelector, RadioSelectorType } from './RadioSelector'
+
+// Workaround for missing addEventListener function in tests when using React Native.
+// This sets the global function to an empty callback, ensuring that the code doesn't break during testing.
+global.addEventListener = () => {}
```

Comment:
Pourquoi c'est nécessaire ?
Je n'ai pas vu un seul endroit où ça l'était.

Reply:
Sans cette ligne tous les tests plantent, j'ai regardé avec lui et il n'y a rien qui m'a sauté aux yeux qu'il aurait pu mal faire. Si jamais t'es inspiré ^^

Reply:
J'ai pas d'illumination, d'autant que je ne sais pas pourquoi ça plante.
Mais comment tu expliques qu'on n'utilise ça nulle part dans le code ?

Reply:
La raison pour laquelle je définis `global.addEventListener` dans le fichier de test est de contourner une limitation dans Jest et jsdom. Dans un environnement de test cette fonction n'est pas définie, ce qui provoque l'erreur : 
`TypeError: globalThis.addEventListener is not a function` 
A vrai dire je ne comprend pas non plus pourquoi j'ai besoin de l'utiliser mais aucun test ne passe dû au hook `useSpaceBarAction` qui utilise `globalThis.addEventListener()`

Reply:
Ok c'est bon j'ai compris alors.

A mon avis le souci vient du fait que tu utilises `useSpaceBarAction` dans un fichier de tests natifs. Or, sur une app native, y'a pas la spacebar.

Donc regarde les endroits où on utilise `useSpaceBarAction` et regarde comment sont faits les tests et l'implémentation de `useSpaceBarAction`